mesh/pmesh.cpp

```diff
@@ -3085,8 +3085,7 @@ int ParMesh::GetSharedFace(int sface) const
 
 int ParMesh::GetNFbyType(FaceType type) const
 {
-   MFEM_VERIFY(have_face_nbr_data,
-               "ExchangeFaceNbrData() should be called before using GetNFbyType");
+   const_cast<ParMesh*>(this)->ExchangeFaceNbrData();
```

Comment:
@YohannDudouit, this `const_cast` is kind of bad, but I'm not sure if there's a better solution… what do you think?

Reply:
This probably was the reason why I only used `MFEM_VERIFY` before, though `ExchangeFaceNbrData` does not really change the `Mesh` but makes it valid, so I think the `const_cast` is okay.